### ISSUE TITLE
Some more icon requests

### ISSUE THREAD

**Issue by skalyan91** [rudrab/Shadow#42]
I have in fact switched to Linux from Mac, and am now using the Shadow icons in Fedora. They are every bit as attractive as I thought they'd be!

I find that I still need the following icons:
* Dwarf Fortress (`dwarffortress.png`)
* Elegance Colors Preferences (`elegance-colors`)
* GNU Octave (`octave-logo.svg`)
* NetHack (`nethack.xpm`)
* Pithos (`pithos`)
* GNOME 2048 (`gnome-2048`)

Thanks!

**Comment by skalyan91** [comment]
Also, XScreensaver (`xscreensaver`)

**Comment by skalyan91** [comment]
Sorry, just a few more!
* Conky Manager (`conky-manager`)
* Krita (`calligrakrita2`)
* Leksah (`leksah`)
* Skychart (`skychart`; just use the `celestia` icon)
* Sublime Text (`/opt/sublime_text/Icon/128x128/sublime-text.png`; somehow, the icon you already have isn't working for me)
* XEphem (`XEphem.png`; again, just use the `celestia` icon)

**Comment by rudrab** [comment]
Hi, 
Thanks again for your request. 
 Since the sublime text is using absolute path,  themed icons won't work. The work around is to copy the  .desktop file to ur local space and edit it. 

Also, please consider sending tips. That is very encouraging. You can send it via PayPal to my email 'bnrj.rudra@live.com'.

**Comment by rudrab** [comment]
Hi,
most of the required icons are added. Please try them.

**Comment by skalyan91** [comment]
Thanks—these look great! (Some of the apps turned out to use absolute paths for icons, so I needed to edit the `.desktop` files; but once I did, the icons worked fine.) I think the “Pithos” icon might need a little more contrast, though (and move rightwards slightly; but it’s your judgment). And I like the new gedit icon!

If you haven’t already started working on the `elegance-colors` icon, don’t worry about it; I probably won’t need it, and I suspect few others will.

I’d like to make just one more request, and then I’ll have everything I need:
* Inform 7 (`inform7`)
